pandas/compat/pickle_compat.py

```diff
@@ -74,9 +74,13 @@ def load_reduce(self):
         ('pandas._libs.sparse', 'BlockIndex'),
     ('pandas.tslib', 'Timestamp'):
         ('pandas._libs.tslib', 'Timestamp'),
+    ('pandas._period', 'Period'): ('pandas._libs.period', 'Period'),
+
+    # 18014 moved __nat_unpickle from _libs.tslib-->_libs.tslibs.nattype
     ('pandas.tslib', '__nat_unpickle'):
         ('pandas._libs.tslib', '__nat_unpickle'),
```

Comment:
you need to change the to location -> ('pandas._libs.tslibs.nattype', '__nat_unpickle')

Reply:
So should there be one entry or two?  I took the "transitive" comment above to mean two.  Are you saying it should just be:

```
('pandas.tslib', '__nat_unpickle'): ('pandas._libs.tslibs.nattype', '__nat_unpickle')
```

or

```
    ('pandas.tslib', '__nat_unpickle'):
        ('pandas._libs.tslibs.nattype', '__nat_unpickle'),
    ('pandas._libs.tslib', '__nat_unpickle'):
        ('pandas._libs.tslibs.nattype', '__nat_unpickle'),
```

or something else?

Reply:
2nd one
should be an easy test 
one works one doesn’t